test/ts-parser.test.js

```diff
@@ -16,12 +16,12 @@ describe('TypeScript Parser Test', function() {
       tsconfig,
     });
     var parsedData = tsParser.parse();
-    expect(parsedData.sections).to.have.length(6);
-    expect(parsedData.constructs).to.have.length(4);
+    expect(parsedData.sections).to.have.length(7);
+    expect(parsedData.constructs).to.have.length(5);
     expect(parsedData.constructs.map(function(c) {
       return c.node.name;
     })).to.eql(['param.Message', 'param.path',
-      'Greeter', 'PathParameterValues']);
+      'Greeter', 'PathParameterValues', 'greeter']);
```

Comment:
Why do we also not check for the namespace `param`?

Reply:
Namespace `param` has become a prefix for its members.

Reply:
So it's impossible to generate docs based on the api docs directly on top of the namespace currently, right?